### ISSUE TITLE
pip package

### ISSUE THREAD

**Issue by blackliner** [truenas/py-libzfs#128]
can you create a pip package?

**Comment by william-gr** [comment]
Not really, it depends on the openzfs version you're using/running. It should be a package on the Linux distro.

**Comment by blackliner** [comment]
How does it depend on the installed ZFS version? All it does is looking for `zfs.h` and compile some stuff? It worked with 0.8 yesterday 🤷 

Or do you mean it **needs** an installed ZFS version, and one would collect those dependencies manually, and thats not piply?
